src/components/TheDestinationPicker.vue

```diff
@@ -1,42 +1,22 @@
 <script setup lang="ts">
 import AutoComplete from "primevue/autocomplete";
 import { TheParametersPicker } from ".";
+import {fetchElement} from "../api/getElement"
 import { ref, watchEffect } from "vue";
 
 const from = ref("");
 const to = ref("");
-
-const createLoop = () => {
-  to.value = from.value;
-};
-
 const cities = ["Gliwice", "Mikołów"];
 const filteredCities = ref();
 
-const emit = defineEmits(["destination-chosen", "destination-not-chosen"]);
-
-const isParamPickerVisible = ref(false);
 
-watchEffect(() => {
-  if (from.value && to.value && from.value.length > 0 && to.value.length > 0) {
-    emit("destination-chosen");
-    isParamPickerVisible.value = true;
-  } else {
-    emit("destination-not-chosen");
-    isParamPickerVisible.value = false;
-  }
-});
+const createLoop = () => {
+  to.value = from.value;
+};
 
-const search = (event: any) => {
-  setTimeout(() => {
-    if (!event.query.trim().length) {
-      filteredCities.value = [...cities];
-    } else {
-      filteredCities.value = cities.filter((city) => {
-        return city.toLowerCase().startsWith(event.query.toLowerCase());
-      });
-    }
-  }, 250);
+const search = async (event: any) => {
+  const data = await fetchElement(event.query,"name",10)
```

Comment:
te 10 wyciągnij proszę do zmiennej i tą zmienną nazwij jakoś żeby określała czym to 10 jest